### ISSUE TITLE
Баги с некоторыми функциями

### ISSUE THREAD

**Issue by Fiftystars** [paulwinex/pw_Houdini_VEX_Editor#1]
Например neighbours(и некоторые другие) выдает:
File "C:/Users/fifty/Documents/houdini15.0/scripts/python\pw_VEX_Editor\widgets\input_widget.py", line 516, in parse_help_line
    hlp = completer_keywords.get_functions_help_window(func)
  File "C:/Users/fifty/Documents/houdini15.0/scripts/python\pw_VEX_Editor\autocomplete\keywords.py", line 48, in get_functions_help_window
    res = r'<br>'.join([r'<u><b>%s</b></u>' % functions_help[func_name].get('hlp','')]+[ x for x in functions_help[func_name]['args']])
KeyError: 'args'

Работать не особо мешает, но все же...

**Comment by paulwinex** [comment]
Да, известная проблема. Почти решена, в новом апдейте не будет её.

**Comment by Vol46** [comment]
Я попробовал убрать эту проблему... Вроде нормально получилось, хотя на питоне никогда не писал. 
Для версии 1.0.7
Прикрепляю модифицированный файл keywords.py
[keywords.zip](https://github.com/paulwinex/pw_Houdini_VEX_Editor/files/514780/keywords.zip)
Его нужно извлечь из архива и поместить по этому пути:
_C:\Users\user_name\Documents\houdini15.5\scripts\python\pw_VEX_Editor\autocomplete\keywords.py_
Затем нужно удалить файлы
_C:\Users\user_name\Documents\houdini15.5\scripts\python\pw_VEX_Editor\autocomplete\keywords.pyc_ и _C:\Users\user_name\Documents\houdini15.5\pw_vex_editor\autocompleter.json_
и при запуске Гудини создаст их заново.

**Comment by paulwinex** [comment]
Спасибо, проверим

7 октября 2016 г., 1:08 пользователь Vol46 notifications@github.com
написал:

**Comment by cspix** [comment]
This keyword.py file didn't work for me so I modified it this way:

```
def get_functions():
    global functions
    if functions:
        return functions

    # There is a bug in the generation of the function cached file
    # and this line bypass that generation. Read from the source
    # all the time. Slower but works.
    return sorted(words['functions_all'], key=lambda x: len(x))

    '''
    cache_file = vex_settings.get_autocomplete_cache_file()
    if os.path.exists(cache_file):
        comp = json.load(open(cache_file))
        functions = sorted(comp['functions'].keys(), key=lambda x: len(x))
        return functions
    else:
        return sorted(words['functions_all'], key=lambda x: len(x))
    '''
```

It basicaly bypass the cache file. There is no speed impact that I can see.
